docs/source/_inc/_branch-policy.md

```diff
@@ -31,11 +31,14 @@ canary
 
     If the pull request is a feature or a bugfix, and if the release manager deems it useful to the latest version's branch, they may ask you to backport it to that branch.
 
-`17.x.x`
-:   `17.x.x` is the current stable and actively developed branch in git.
-    Upon the final release of version `18.0.0`, the `17.x.x` branch line will be no longer supported.
+`18.x.x`
+:   `18.x.x` is the current stable and actively developed branch in git.
```

Comment:
Are you referring if it would be LTS as well? We decided kind of "drop" the term. 
In fact, in  https://plone.org/download/release-schedule there is no mention at all of it.
But yes, let me amend it to reflect it.

Reply:
@stevepiercy I just updated it, please take a look.

Reply:
This one looks good.